homeassistant/components/tcp/sensor.py

```diff
@@ -72,6 +79,15 @@ def __init__(self, hass, config):
             CONF_VALUE_ON: config.get(CONF_VALUE_ON),
             CONF_BUFFER_SIZE: config.get(CONF_BUFFER_SIZE),
         }
+
+        if config.get(CONF_SSL):
+            self._ssl_context = ssl.create_default_context()
+            if not config.get(CONF_VERIFY_SSL):
```

Comment:
```suggestion
            if not config[CONF_VERIFY_SSL]:
```
`get()` is not needed, because `config[CONF_VERIFY_SSL]` always contains a value (_at least by `default=DEFAULT_VERIFY_SSL`_)